korge/src/commonMain/kotlin/com/soywiz/korge/view/tiles/TileMap.kt

```diff
@@ -311,7 +326,7 @@ open class TileMap(
 
         infos.fastForEach { buffer ->
             ctx.batch.drawVertices(
-                buffer.vertices, ctx.getTex(buffer.tex), smoothing, renderBlendMode.factors, buffer.vcount, buffer.icount
+                buffer.vertices, ctx.getTex(buffer.tex), smoothing, AG.Blending(AG.BlendFactor.ONE, AG.BlendFactor.ONE_MINUS_SOURCE_ALPHA), buffer.vcount, buffer.icount
```

Comment:
If this is not working, this should be a separate issue. Maybe related to this? https://github.com/korlibs/korge/issues/95

That's usually related to premultiplied alpha. I have found in some cases PNG images that had alpha premultiplied against the specification. There are ways to asume it as premultiplied alpha like here: https://github.com/korlibs/korge-samples/blob/b3f007ee30c51c827bb9443898b778778a6d1c50/samples/spine/src/commonMain/kotlin/mainCommon.kt#L12

Also calling `AG.Blending` performs an allocation, so at least it should be cached. But I believe it should be handled elsewhere.

Can you remove this line for this PR and provide a tilemap/tileset reproducing your issue so we can investigate?

Reply:
Sounds good! I'll put together a sample project with the tileset to share.

Reply:
Thanks! Once we have a sample that we can use, we could also add tests verifying that everything works well (including ortogonal, staggered and plain isometric when supported), so we can refactor safely later